packages/ffe-form-react/src/InputGroup.spec.js

```diff
@@ -21,7 +21,7 @@ describe('<InputGroup>', () => {
 
     it('renders the given child', () => {
         const wrapper = getWrapper();
-        expect(wrapper.find('Input')).toHaveLength(1);
+        expect(wrapper.find(Input)).toHaveLength(1);
```

Comment:
`expect(wrapper.find(Input).exists()).to.be.true;` eller vill du vare sikker på at det er fler en en?